samples/DirectX/D3D12/HelloConstBuffer12.cs

```diff
@@ -0,0 +1,807 @@
+                }
+
+                // Describe and create the swap chain.
+                var swapChainDesc = new DXGI_SWAP_CHAIN_DESC1 {
+                    BufferCount = FrameCount,
+                    Width = Width,
+                    Height = Height,
+                    Format = DXGI_FORMAT_R8G8B8A8_UNORM,
+                    BufferUsage = DXGI_USAGE_RENDER_TARGET_OUTPUT,
+                    SwapEffect = DXGI_SWAP_EFFECT_FLIP_DISCARD,
+                    SampleDesc = new DXGI_SAMPLE_DESC(count: 1, quality: 0),
+                };
+
+                ThrowIfFailed(nameof(IDXGIFactory4.CreateSwapChainForHwnd), factory->CreateSwapChainForHwnd(
+                    (IUnknown*)_commandQueue,         // Swap chain needs the queue so that it can force a flush on it.
+                    Win32Application.Hwnd,
+                    &swapChainDesc,
+                    pFullscreenDesc: null,
+                    pRestrictToOutput: null,
+                    &swapChain
+                ));
+
+                // This sample does not support fullscreen transitions.
+                ThrowIfFailed(nameof(IDXGIFactory4.MakeWindowAssociation), factory->MakeWindowAssociation(Win32Application.Hwnd, DXGI_MWA_NO_ALT_ENTER));
+
+                fixed (IDXGISwapChain3** swapChain3 = &_swapChain)
+                {
+                    iid = IID_IDXGISwapChain3;
+                    ThrowIfFailed(nameof(IDXGISwapChain1.QueryInterface), swapChain->QueryInterface(&iid, (void**)swapChain3));
+                    _frameIndex = _swapChain->GetCurrentBackBufferIndex();
+                }
+
+                // Create descriptor heaps.
+                {
+                    // Describe and create a render target view (RTV) descriptor heap.
+                    var rtvHeapDesc = new D3D12_DESCRIPTOR_HEAP_DESC {
+                        NumDescriptors = FrameCount,
+                        Type = D3D12_DESCRIPTOR_HEAP_TYPE_RTV,
+                    };
+
+                    fixed (ID3D12DescriptorHeap** rtvHeap = &_rtvHeap)
+                    {
+                        iid = IID_ID3D12DescriptorHeap;
+                        ThrowIfFailed(nameof(ID3D12Device.CreateDescriptorHeap), _device->CreateDescriptorHeap(&rtvHeapDesc, &iid, (void**)rtvHeap));
+                    }
+
+                    _rtvDescriptorSize = _device->GetDescriptorHandleIncrementSize(D3D12_DESCRIPTOR_HEAP_TYPE_RTV);
+
+                    // Describe and create a constant buffer view (CBV) descriptor heap.
+                    // Flags indicate that this descriptor heap can be bound to the pipeline 
+                    // and that descriptors contained in it can be referenced by a root table.
+                    var cbvHeapDesc = new D3D12_DESCRIPTOR_HEAP_DESC {
+                        NumDescriptors = 1,
+                        Flags = D3D12_DESCRIPTOR_HEAP_FLAGS.D3D12_DESCRIPTOR_HEAP_FLAG_SHADER_VISIBLE,
```

Comment:
Pattern is to `using static TerraFX.Interop.D3D12_DESCRIPTOR_HEAP_FLAGS` and then make this just
```suggestion
                        Flags = D3D12_DESCRIPTOR_HEAP_FLAG_SHADER_VISIBLE,
```